### ISSUE TITLE
winapp2 restoration project phase 2

### ISSUE THREAD

**Issue by MoscaDotTo** [MoscaDotTo/Winapp2#897]
I've done a substantial amount of work in identifying applications that are either of the type that you would want to keep up to date (such as web browsers) that have been discontinued or applications whose vendor no longer offers the ability to download them after discontinuing support, or whose functionality relied on access to services no longer available. I did not remove applications that could just be feature complete and old. 

one thing that has been lingering as i looked up the current state of applications is that many of them actually are under active development, and it makes me wonder whether or not our entries contain accurate keys for the current state of these applications 

So I think we should do an audit, and assess the accuracy of entries for applications who have been updated or released recently. im going to peg "recently" on the grand scale of this project and the content of this file as the last 7 years. 

We should start by collecting a list of the applications in winapp2.ini for applications released or updated since at least 2015 along with links to download them. This is when windows 10 came out so I think it sets a good precedent for what would constitute a "modern" version of an application. 

This may also be a good opportunity to catch any discontinued softwares that may have fallen through the cracks in the refactor. 

Ideally, a link to download the software from the original vendor or from a trusted third party can be provided. we should take note of software for which a safe link is not easily found as well.

We can omit games, lets include products that are not free for now and we can figure that out when we get to it 

.NET Reflector: https://www.red-gate.com/products/dotnet-development/reflector/

1Click DVDCopy Pro http://www.lgsoftwareinnovations.com/index.asp

etc 

We should start this after phase 1 (file size squeeze) is complete

**Comment by APMichael** [comment]
Good idea, I really hope there are some people in the community who want to help with this. (Unfortunately, the community is currently very quiet, except for the 3-4 people who regularly post new or updated entries).
